Rocket.Chat/BlockedUsersListViewController.swift

```diff
@@ -0,0 +1,142 @@
+            cell.data = MemberCellData(member: user)
+            return cell
+        }
+
+        return UITableViewCell()
+    }
+
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return memberCellHeight
+    }
+}
+
+// MARK: TableViewDelegate
+
+extension BlockedUsersListViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: true)
+        guard let user = findUser(userIdentifire: MessageManager.blockedUsersList[indexPath.row], realm: self.realm) else { return }
+        guard let username = user.username else { return }
+        blockedUserActions(username, user)
+    }
+
+    func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCellEditingStyle, forRowAt indexPath: IndexPath) {
+        guard let user = findUser(userIdentifire: MessageManager.blockedUsersList[indexPath.row], realm: self.realm) else { return }
+        if editingStyle == .delete {
+            MessageManager.unblockMessagesFrom(user, completion: {
+                UIView.performWithoutAnimation {
+                    self.manageEmptyView()
+                    self.membersTableView?.reloadData()
+                }
+            })
+        }
+    }
+}
+
+// MARK: Blocked users' actions
+
+extension BlockedUsersListViewController {
+    func blockedUserActions(_ username: String, _ user: User) {
+        if AuthManager.isAuthenticated()?.canUnblockUser(user) == .allowed {
+            let alert = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
+            alert.addAction(UIAlertAction(title: localized("chat.member.open.profile.title"), style: .default, handler: { _ in
+                AppManager.openDirectMessage(username: username) {
+                    self.dismiss(animated: true, completion: nil)
+                }
+            }))
+            alert.addAction(UIAlertAction(title: localized("chat.member.unblock.title"), style: .destructive, handler: { _ in
+                MessageManager.unblockMessagesFrom(user, completion: {
+                    UIView.performWithoutAnimation {
+                        self.manageEmptyView()
+                        self.membersTableView?.reloadData()
+                    }
+                })
+            }))
+            alert.addAction(UIAlertAction(title: localized("global.cancel"), style: .cancel, handler: nil))
+
+            if let presenter = alert.popoverPresentationController {
+                presenter.sourceView = view
+                presenter.sourceRect = view.bounds
+            }
+            present(alert, animated: true, completion: nil)
+        }
+    }
+
+    func findUser(userIdentifire: String, realm: Realm? = Realm.shared) -> User! {
+        guard
+            let realm = realm,
+            let user = realm.object(ofType: User.self, forPrimaryKey: userIdentifire)
+            else {
```

Comment:
Can you move this else one indentation back? (should be the same level as guard)